apps/taiga-lumbermill/src/dashboards/crypto/components/nft/nft.service.ts

```diff
@@ -0,0 +1,62 @@
+import {Injectable} from '@angular/core';
+
+export interface NFT {
```

Comment:
let's add field `readonly transactions: Transaction[]` so each NFTT has it's own transaction